### PR TITLE
EN 964 labels search form

### PR DESCRIPTION
**Pull request by mcarta-rt**

### CODE REVIEW

**Comment by entando-service** [comment]
All is well

File: src/ui/labels/list/LabelSearchForm.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    ev.preventDefault();
+    handleSubmit();
+  };
+
+  return (
+    <form onSubmit={onSubmit} className="LabelSearchForm form-horizontal well">
+      <h3><FormattedMessage id="label.searchForm.title" /></h3>
+      <FormGroup>
+        <Row>
+          <label className="control-label col-sm-2" htmlFor="username">
```
Comment (fedelocci) [inline src/ui/labels/list/LabelSearchForm.js]: col-xs

File: src/ui/labels/list/LabelSearchForm.js [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+            <FormattedMessage id="label.searchForm.text" />
+          </label>
+          <Col sm={9}>
```
Comment (fedelocci) [inline src/ui/labels/list/LabelSearchForm.js]: column xs

File: src/ui/labels/list/LabelsAndLanguagesPage.js [hunk 3]
```diff
@@ -38,7 +39,37 @@ class LabelsAndLanguagesPage extends Component {
...
+      pageContent = (
+        <Row>
+          <Col xs={12}>
+            <Row>
+              <Col xs={6} xsOffset={3}>
+                <LabelSearchFormContainer />
+              </Col>
+            </Row>
+            <Row>
+              <Col md={12}>
```
Comment (fedelocci) [inline src/ui/labels/list/LabelsAndLanguagesPage.js]: col xs

**Comment by entando-service** [comment]
All is well

**Pull request merged**